content/hardware/05.pro-solutions/solutions-and-kits/opta/tutorials/getting-started-connectivity/content.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Using BLE, WiFi and Ethernet on the Arduino Opta'
+description: "Learn how to make use of the Arduino Opta's connectivity features"
+difficulty: beginner
+tags:
+  - Wi-Fi®
+  - Ethernet
+  - Bluetooth®
+  - Connectivity
+author: 'Benjamin Dannegård'
+libraries:
+  - name: Arduino BLE
+    url: https://www.arduino.cc/reference/en/libraries/arduinoble/
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+hardware:
+  - hardware/05.pro-solutions/solutions-and-kits/opta
+---
+
+## Overview
+
+The Arduino® Opta is a powerful PLC device that has many features, allowing you to customize its use for your solution. Among these features are the standard connectivity features: Wi-Fi®, Ethernet and Bluetooth®. In this tutorial we will go through how to use these features with the Arduino IDE and the Arduino Opta.
+
+## Goals
+
+- Learn how to use Bluetooth® Low Energy on the Opta
+- Learn how to use Wi-Fi® on the Opta
+- Learn how to use Ethernet on the Opta
+
+### Required Hardware and Software
+
+- Arduino Opta
+- Ethernet cable
+- [Arduino IDE](https://www.arduino.cc/en/software)
+- USB-C® cable (either USB-C to USB-A or USB-C to USB-C)
+- Wi-Fi® Access Point with Internet Access
+
+## Instructions
+
+Using the Arduino IDE we can easily work with these peripherals. Some features have their own library that we can make use of. First we need to install the appropriate core for the Arduino Opta, go into the **Board manager** and search for **Opta Mbed core**. When the latest version of the core is installed, we can move on to try out the connectivity options.
```

Comment:
Here it would be nice to have a small section (As the Getting Started with Modbus RTU and RS485 Tutorial has) of environment setup (Arduino IDE setup). In my humble opinion, this would help to guide user more friendly although it is already mentioned how to do in other tutorials. Redundancy sometimes helps a lot.

Reply:
I agree, redundancy is a powerful tool but maintenance it's a dangerous long term problem .

I'm tempted to just copy and paste the content related from the getting started but... if in the future we have to update something... we will probably miss to change it here too.

Let me think about it and reach for this case a compromise solution.

@BenjaminDannegard please add a line similar to " First we need to install the appropriate core for the Arduino Opta, go into the **Board manager** and search for **Opta Mbed core**. If you need more info about it, please take a look at (getting started link)"

PD: Don't be afraid of redundancy redundancy is good

PD2: Don't fear redundancy, it's a powerful tool

PD3: I don't know if I mentioned this before, but redundancy as Taddy says could be great